docs/adding_tracks.rst

```diff
@@ -626,6 +626,7 @@ Template Language
 
 Rally uses `Jinja2 <http://jinja.pocoo.org/docs/dev/>`_ as a template language so you can use Jinja2 expressions in track files.
 
+Elasticsearch utilizes Mustache formatting in a few places, notably in `Search templates <https://www.elastic.co/guide/en/elasticsearch/reference/7.4/search-template.html>`_ and `Watcher templates <https://www.elastic.co/guide/en/elasticsearch/reference/7.4/actions-email.html>`_. If you are using Mustache in your Rally templates you must `escape them properly <https://jinja.palletsprojects.com/en/2.10.x/templates/#escaping>`_. See :ref:`put_pipeline` for an example.
```

Comment:
Any reason you've removed the `.. note::` that you've added in the previous commit?

Reply:
Nit: "Search templates" -> "search templates"

Reply:
ya i removed it cuz it got big enough for a full paragraph and just looked like a huge note w/ a bunch of links, rather than some concise text. I can change it back tho if youd like